packages/lodestar-types/src/ssz/interface.ts

```diff
@@ -20,8 +20,8 @@ export interface IBeaconSSZTypes {
   Uint64: BigIntUintType;
   Uint128: BigIntUintType;
   Uint256: BigIntUintType;
-  Slot: NumberUintType;
-  Epoch: NumberUintType;
+  Slot: BigIntUintType;
+  Epoch: BigIntUintType;
   CommitteeIndex: NumberUintType;
   ValidatorIndex: NumberUintType;
```

Comment:
Both CommitteeIndex and ValidatorIndex should be BitIntUintType.
One of the goals of this change was to prevent api response discrepancies ( we serialize bigints as string in json while numbers remain native)